src/sentry/testutils/helpers/api_gateway.py

```diff
@@ -45,9 +67,14 @@ def verify_request_params(params, headers):
 
     def request_callback(request):
         request_params = parse_qs(request.url.split("?")[1])
-        assert request_params == params
         assert (request.headers[key] == headers[key] for key in headers)
-        return 200, {}, b"{}"
+        for key in params:
+            assert key in request_params
+            if len(request_params[key]) > 1:
+                assert request_params[key] == params[key]
+            else:
+                assert request_params[key][0] == params[key]
```

Comment:
parse_qs returns all parameters as lists, even singletons. So `?foo=bar` becomes `{"foo":["bar"]}`. As such we want to compare the 0th index in the list to the original param.

List query parameters are not wrapped in another list